### DISCUSSION TITLE
Про mso_default_head_section()

### DISCUSSION THREAD

**Discussion by nicothin** [maxsite/cms#429] [Q&A]
Привет.
Работаю с проектом, который сделал в 2012 году (с тех пор ведущие проекта ничего программно с ним не делали). Обновил версию движка, сломался шаблон (не выводится ничего до `body`). 
Кажется, проблема в `mso_default_head_section()`, что я использовал в шаблоне в 2012 году.

Вопрос: как быть со старым шаблоном?
Насколько вижу, `mso_default_head_section()` теперь отсутствует в движке. Видимо, мне нужно подменить ее на уровне шаблона, вопрос, скорее, в том, откуда её взять в работающем виде. 

Простое копирование этой функции из файлового бекапа в main.php моего шаблона приводит к тому, что не работает фрагмент с подключением файлов плагинов (не подключается jQuery, не работает слайдер и, возможно, что-то еще).

Что приходится закомментировать:
```
...
		// out_component_css();
		// echo NT . mso_load_jquery();
		// mso_hook('head');
		// default_out_profiles();
...
```

Может, я не туда копирую этот самый `mso_default_head_section()` ?

**Comment by maxsite** [comment]
Проще взять новый default и на него натянуть нужный дизайн. Внутренне шаблоны сильно изменились за эти 9 лет.
